src/D_EcobeeHouse1.json

```diff
@@ -35,10 +35,10 @@
     ],
     "inScene": "1",
     "DisplayStatus": {
-        "Service": "urn:upnp-org:serviceId:SwitchPower1",
-        "Variable": "Status",
-        "MinValue": "0",
-        "MaxValue": "1"
+        "Service": "urn:ecobee-com:serviceId:Ecobee1",
+        "Variable": "currentClimateRef",
+        "MinValue": "home",
+        "MaxValue": "smart1"
```

Comment:
Please update the doc on the `gh-pages` branch with a good explanation of the specific meaning you are attaching to the dynamically assigned `smart1` climate ref.  From my understanding `smartX` is generated by the API when users create new climates, and `smart1` might get assigned to a climate whose purpose is different from what you might be assuming.  Therefore, hardcoding `smart1` in the plugin might lead to issues who use a user-defined climate differently than how you expect.